integration/query_comparision_test.go

```diff
@@ -260,7 +260,18 @@ func TestQueryComparisonEq(t *testing.T) {
 			filter:      bson.D{{"value", bson.D{{"$eq", int64(math.MinInt64)}}}},
 			expectedIDs: []any{"int64-min"},
 		},
-
+		"Float64AndInt64": {
+			filter:      bson.D{{"value", bson.D{{"$eq", float64(2 << 61)}}}},
+			expectedIDs: []any{"int64-big"},
+		},
+		"Int64AndFloat64": {
+			filter:      bson.D{{"value", bson.D{{"$eq", int64(2 << 60)}}}},
+			expectedIDs: []any{"float64-big"},
+		},
+		"Float32AndInt32": {
+			filter:      bson.D{{"value", bson.D{{"$eq", float32(2 << 25)}}}},
+			expectedIDs: []any{"int32-big"},
+		},
```

Comment:
Notice how before that change tests cases were grouped by argument type and separated by blank lines. We should keep that organization.

Rename test cases based on argument type and move them to existing "Double...", "Int32..." and "Int64..." groups.

Reply:
I agree, fixed